src/ivc/incrementally_verifiable_computation.rs

```diff
@@ -53,8 +66,6 @@ where
     primary: StepCircuitContext<A1, C1, SC1>,
     secondary: StepCircuitContext<A2, C2, SC2>,
 
-    primary_trace: PlonkTrace<C2>,
```

Comment:
We still need `primary_trace` to be part of IVC. With the current design of IVC circuit, it is not used at step=0, we will need it for step>0.
To keep the same process as Nova, we can define it as:
```
pub struct IVC {
  primary:StepCircuitContext, // U1
 secondary: StepCircuitContext, // U2
 secondary_trace: PlonkTrace, // u2
 step: usize,
}
```
With this definition, we will (1) fold the u2 and U2 from secondary circuit using primary circuit (2) fold u1 (generated from step 1) and U1 using secondary circuit.

Reply:
I know, but due to the new nifs design, I've removed that field for now and decided to keep it while writing the fold step

Reply:
Before that we had to store `TableData`, reminder